R/plotNorm.R

```diff
@@ -76,25 +81,137 @@ plotNorm <- function(DGEdata,
 
     tall <- tall %>%
         rbind(tall_tmm)
+    title <- stringr::str_c("Log2CPM before/after", normalize, "normalization", sep = " ")
 
-    if (tolower(plotType) == "density") {
-        resultPlot <- ggplot(tall, aes(x = Log2CPM, color = SampleID)) +
-            geom_density() +
-            facet_grid(~Normalization) +
-            ggtitle(stringr::str_c("Log2CPM before/after", normalize, "normalization", sep = " "))  +
-            theme_gray() +
-            theme(legend.position = "none")
+    if (tolower(plotType) == "canvasxpress") {
```

Comment:
We already have assert statement for checking the exact case for plotType. You can remove tolower() for plotype